src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -579,7 +576,7 @@ export class NotebookService extends Disposable implements INotebookService {
 		}
 	}
 
-	serializeNotebookStateChange(notebookUri: URI, changeType: NotebookChangeType, cell?: ICellModel): void {
+	async serializeNotebookStateChange(notebookUri: URI, changeType: NotebookChangeType, cell?: ICellModel, isTrusted?: boolean): Promise<void> {
```

Comment:
This was changed to return a promise since we now have to wait on the method that updates the trusted cache. Without it, the new editor could open before the trusted state is updated, so it would appear as untrusted even though it isn't in the cache.